.github/workflows/cosmos-compatibility.yml

```
@@ -0,0 +1,277 @@
+      - '.github/workflows/cosmos-compatibility.yml'
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  abci-compatibility:
+    name: ABCI++ Protocol Compatibility
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)